qwazzock/server.py

```diff
@@ -71,6 +71,13 @@ def host_client_socket_standard_event():
         game.question_type = "standard"
         update_clients()
 
+    @socketio.on("team", namespace="/host_client_socket")
+    def host_client_socket_team_event(json):
```

Comment:
Is this method solely responsible for handling an event where a team should be locked out?  If so, could be named something clearer - host_client_socket_team_lockout_event(json) ?

Reply:
At present yes, but its highly likely the event will be shared for other "do something to a team" actions based on context, so I figured I'd leave it vague.